projecttacoma/synthea
# Rebase abacus dev

# Summary
Rebasing our fork with the most recent version of synthetichealth/synthea

## New Behavior
Bug fixes and new behavior introduced by synthetichealth since our last rebase

## Code Changes
simple rebase

# Testing Guidance
 - Run the unit tests using `./gradlew build check test`. This took about 13 minutes on my machine and produced 103 warnings, which was equivalent with synthetichealth/synthea:master. 
 - Download the .sh file posted in Slack into the same directory as your synthea repo
 - Navigate to that repo and run `chmod +x synthea-test.sh`
 - Then run `./synthea-test.sh`
 - Ensure no errors occur while the file runs. You should see build successful after each command runs
 - If this looks good don't merge! Instead I will force push my local abacus-dev branch. This is the only way to preserve all commits and avoid merge conflicts
